library/build.gradle

```diff
@@ -5,17 +5,17 @@ repositories {
 }
 
 dependencies {
-    compile 'com.android.support:support-v4:23.4.0'
-    compile 'com.android.support:support-annotations:23.4.0'
-    compile 'com.android.support:recyclerview-v7:23.4.0'
+   compile 'com.android.support:support-v4:26.0.0-alpha1'
```

Comment:
I prefer to avoid alpha libraries. If you change this to a stable version like 25, i'll merge. Thank you!